controller.py

```diff
@@ -0,0 +1,112 @@
+from flask import Flask, jsonify, request
+from cmd_bd import select
+from serializers import diretores_from_db, usuario_from_web, usuario_from_db
+from serializers import genero_from_db, genero_from_web, diretores_from_web, nome_usuario_from_web
+from models import insert_diretor, update_diretor, delete_genero, update_usuario, delete_usuario,delete_diretor
+from models import get_genero, insert_genero, get_diretor, get_usuario, insert_usuario, select_usuarios
+from validacao import valida_genero, valida_diretores, valida_usuario
+
+app = Flask(__name__)
+
+
+@app.route("/diretores", methods=["POST"])
+def inserir_diretores():
+    diretor = diretores_from_web(**request.json)
+    if valida_diretores(**diretor):
+        insert_diretor(**diretor)
+        diretor_criado = get_diretor(diretor["nome_completo"])
```

Comment:
```suggestion
        id_diretor = insert_diretor(**diretor)
        diretor_criado = get_diretor(id_diretor)
```
Lembra que agora o select retorna o id